File: swagger/models/get_client_referral_types_request.py
from .base_model import BaseModel


class GetClientReferralTypesRequest(BaseModel):
	"""
	no description available

	Attributes:

		include_inactive
		When `true`, filters the results to include subtypes and inactive
		referral types.<br />  When `false`, includes no subtypes and only
		active types.
	"""

	input_map = {
		'IncludeInactive': 'include_inactive',
		}

	output_map = {
		'include_inactive': 'IncludeInactive',
		}

File: swagger/Models/GetClientReferralTypesRequest.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetClientReferralTypesRequest extends BaseModel
{
	/**
	 * When `true`, filters the results to include subtypes and inactive
	 * referral types.<br />  When `false`, includes no subtypes and only
	 * active types.
	 * 
	 * @var boolean 
	 */
	public $includeInactive;

	protected function getInputMap()
	{
		return [
			'IncludeInactive' => ['includeInactive', 'boolean', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'includeInactive' => ['IncludeInactive', 'boolean', null],
			];
	}
}
